macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -324,12 +324,25 @@ open class AvatarView: NSView {
 	/// - returns: the color table entry for the given index
 	///
 	/// - note: Internal visibility exists only for unit testing
-	@objc(getcolorForIndex:)
+	@objc(getColorForIndex:)
 	public static func getColor(for index: Int) -> ColorSet {
 		let avatarBackgroundColors = AvatarView.avatarColors
 		return avatarBackgroundColors[index % avatarBackgroundColors.count]
 	}
 
+	/// Get the legacy color associated with a given index
+	/// - parameter index: the index into the color table
+	///
+	/// - returns: the color table entry for the given index
+	///
+	/// - note: API method avaiable for legacy support only, This will be removed in future release
+	@objc(getLegacyBackgroundColorForIndex:)
+	@available(*, deprecated, message: "Use getColorForIndex instead")
+	public static func getLegacyColor(for index: Int) -> NSColor {
+		let avatarBackgroundColors = AvatarView.legacyAvatarViewBackgroundColor
```

Comment:
nit: can we just rename the variable to be "legacy" as well? (avoiding confusion with the implementation above)